src/theme/material/select.m.css

```diff
@@ -28,7 +28,6 @@
 
 .disabled {
 	composes: mdc-select--disabled from '@material/select/dist/mdc.select.css';
-	background-color: var(--mdc-disabled-color);
```

Comment:
I wasn't sure why we ever had this to begin with, as `mdc-select--disabled` provides the correct disabled background already, which is the same as `Input`.